scripts/ebtc_governance.py

```diff
@@ -0,0 +1,220 @@
+    safe.init_ebtc()
+    C.print(f"\nUsing {safe.account} for CDPManager.setGracePeriod\n")
+    safe.ebtc.cdpManager_set_grace_period(value)
+
+    safe.post_safe_tx()
+
+
+def priceFeed_set_fallback_caller(address):
+    """
+    Set the fallback caller for the PriceFeed contract.
+    """
+    safe = GreatApeSafe(r.ebtc_wallets.techops_multisig)
+    safe.init_ebtc()
+    C.print(f"\nUsing {safe.account} for PriceFeed.setFallbackCaller\n")
+    safe.ebtc.priceFeed_set_fallback_caller(address)
+
+    safe.post_safe_tx()
+
+
+def activePool_set_fee_bps(value):
+    """
+    Set the fee basis points for the ActivePool contract.
+    """
+    safe = GreatApeSafe(r.ebtc_wallets.techops_multisig)
+    safe.init_ebtc()
+    C.print(f"\nUsing {safe.account} for ActivePool.setFeeBps\n")
+    safe.ebtc.activePool_set_fee_bps(value)
+
+    safe.post_safe_tx()
+
+
+def borrowerOperations_set_fee_bps(value):
+    """
+    Set the fee basis points for the BorrowerOperations contract.
+    """
+    safe = GreatApeSafe(r.ebtc_wallets.techops_multisig)
+    safe.init_ebtc()
+    C.print(f"\nUsing {safe.account} for BorrowerOperations.setFeeBps\n")
+    safe.ebtc.borrowerOperations_set_fee_bps(value)
+
+    safe.post_safe_tx()
+
+
+def activePool_set_fee_recipient_address(address):
+    """
+    Set the fee recipient address for the ActivePool contract.
+    """
+    safe = GreatApeSafe(r.ebtc_wallets.techops_multisig)
+    safe.init_ebtc()
+    C.print(f"\nUsing {safe.account} for ActivePool.setFeeRecipientAddress\n")
+    safe.ebtc.activePool_set_fee_recipient_address(address)
+
+    safe.post_safe_tx()
+
+
+"""
+High Sec Governance Operations
+
+The following operations are meant to be executed by the Ecosystem multisig.
+"""
+
+
+def authority_set_role_name(role, name):
+    """
+    Set the role name for the Authority contract.
+    """
+    safe = GreatApeSafe(r.ebtc_wallets.ecosystem_multisig)
+    safe.init_ebtc()
+    C.print(f"\nUsing {safe.account} for Authority.setRoleName\n")
+    safe.ebtc.authority_set_role_name(role, name)
+
+    safe.post_safe_tx()
+
+
+def authority_set_user_role(user, role, enabled):
+    """
+    Set the user role for the Authority contract.
+    """
+    safe = GreatApeSafe(r.ebtc_wallets.ecosystem_multisig)
+    safe.init_ebtc()
+    C.print(f"\nUsing {safe.account} for Authority.setUserRole\n")
+    safe.ebtc.authority_set_user_role(user, role, enabled)
+
+    safe.post_safe_tx()
+
+
+def authority_set_role_capability(role, target_address, functionSig, enabled):
+    """
+    Set the role capability for the Authority contract.
+    """
+    safe = GreatApeSafe(r.ebtc_wallets.ecosystem_multisig)
+    safe.init_ebtc()
+    C.print(f"\nUsing {safe.account} for Authority.setRoleCapability\n")
+    safe.ebtc.authority_set_role_capability(role, target_address, functionSig, enabled)
+
+    safe.post_safe_tx()
+
+
+def authority_set_public_capability(target_address, functionSig, enabled):
```

Comment:
Another element that may help while setting role capabilities is to have something similar to the address book, the “signatures” book for signatures such as: mint, burn, set staking reward etc

Basically those which are expected to be used to decrease the probability of failure when calling the scripts, having something like a dictionary where `DICTIONARY_SIGNATURES[“MINT”]` -> ‘0x…..’ may be handy imo

Reply:
resolved https://github.com/ebtc-protocol/ebtc-multisig/pull/3/commits/f4f448d10451d3a2ea524233cd8ae229b7dd55d1